sign/bls/bls.go

```diff
@@ -2,7 +2,7 @@
 // was introduced in the paper "Short Signatures from the Weil Pairing". BLS
 // requires pairing-based cryptography.
 //
-// Deprecated: This version is vulnerable to rogue public-key attack and the
+// This version is vulnerable to rogue public-key attack and the
```

Comment:
Maybe specify it's in case of aggregation only and that regular BLS signatures are fine